boards.txt

```diff
@@ -228,7 +228,11 @@ Generic_nRF51822.build.board=GENERIC
 Generic_nRF51822.build.core=nRF5
 Generic_nRF51822.build.variant=Generic
 Generic_nRF51822.build.variant_system_lib=
+<<<<<<< Updated upstream
+Generic_nRF51822.build.extra_flags=-DNRF51 -DNRF51822
+=======
 Generic_nRF51822.build.extra_flags=-DNRF51
+>>>>>>> Stashed changes
```

Comment:
Yes, if they are already labelled in the board with the nrf pin, then is better to keep the nrf and arduino pin numbering the same.
Then I believe -1 is the value used in other variants for that.